salt/output/highstate.py

```diff
@@ -135,6 +135,8 @@ def _format_host(host, data):
 
             tcolor = colors['GREEN']
             schanged, ctext = _format_changes(ret['changes'])
+            if not ctext and 'pchanges' in ret:
+                schanged, ctext = _format_changes(ret['pchanges'])
```

Comment:
On line  276 below is this line:
```
changes = u'     Changes:   ' + ctext
```
To help differentiate between changes and pchanges, it might be nice to hoist that line up here to be in the `else` of this block, and add a line in this if block to set changes to something like
```
changes = u'    PChanges:   ' + ctext
```

It looks like multiple colors are already used in the changes section, so I don't think we need to do much there.